app/src/main/res/values/styles.xml

```diff
@@ -362,7 +362,7 @@
         <item name="android:contentDescription">@null</item>
         <item name="placeholderImage">?attr/material_theme_border_color</item>
         <item name="failureImage">?attr/material_theme_border_color</item>
-        <item name="backgroundImage">?attr/material_theme_border_color</item>
+        <item name="backgroundImage">?attr/paper_color</item>
```

Comment:
It doesn't seem like `paper_color` is the right choice here. Robin seems to be asking for a variation of `paper_color` that is "shaded" with an opacity. He will need to invent a new meta-color for this purpose.

Reply:
OK! This fixes the color of placeholder thumbnails in the composite (2x2) thumbnail, but it also breaks the color of thumbnails of articles that don't have an image. For example:
![device-2019-03-12-161225](https://user-images.githubusercontent.com/1682214/54232758-d39b2000-44e1-11e9-814a-b481f9493c91.png)

Reply:
done!